cedar-lean/Cedar/Validation/Types.lean

```diff
@@ -55,6 +55,7 @@ inductive CedarType where
   | entity (ety : EntityType)
   | set (ty : CedarType)
   | record (rty : Map Attr (Qualified CedarType))
+  | attribute_map (aty : CedarType)
```

Comment:
Can we rename this to `attributeMap`?  

We use the Mathlib / Lean naming convention where snake_case is reserved for theorem names.

See https://github.com/cedar-policy/cedar-spec/blob/main/cedar-lean/GUIDE.md#casing